README.md

```diff
@@ -1,6 +1,6 @@
 # boot-sassc
 
-[![Clojars Project](http://clojars.org/mathias/boot-sassc/latest-version.svg)](http://clojars.org/mathias/boot-sassc)
+[![Clojars Project](http://clojars.org/boot-sassc/latest-version.svg)](http://clojars.org/mathias/boot-sassc)
```

Comment:
I think we want to link to https://clojars.org/boot-sassc since that's the latest version, 0.1.2 -- I'll have to see about getting `mathias/boot-sassc` taken off of clojars for less confusion.